test-cli/src/bin/main.rs

```diff
@@ -47,6 +47,10 @@ pub enum Command {
         /// Allow partial fills
         #[clap(long)]
         allow_partial_fills: bool,
+
+        /// Path to ledgerdb used by the server
+        #[clap(long, env = "MC_LEDGER_DB")]
+        ledger_db_path: PathBuf,
```

Comment:
The server and other parts of the codebase use `--ledger-db`, we should be consistent. 
You can use `long = "ledger-db"`, or just change the var name to `ledger_db`, it's fine if on line 104 the variable gets aliased (I think the compiler would be okay with that)